content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1311,6 +1326,8 @@ Defines a multi-field ID (composite ID) on the model.
 | Name     | Required | Type               | Description                                                      |
 | -------- | -------- | ------------------ | ---------------------------------------------------------------- |
 | `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]` |
+| `fields` | **No**   | `String`           |                                                                  |
+| `fields` | **No**   | `String`           |                                                                  |
```

Comment:
This seems obsolete/leftover?

Reply:
Thanks, copy paste error where I missed updating the argument names